website_sale_invoice_address/static/src/js/website_sale_invoice_address_tour.js

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2020 Tecnativa - David Vidal
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define("website_sale_invoice_address.tour", function (require) {
+    "use strict";
+
+    var tour = require("web_tour.tour");
+
+    var steps = [
+        {
+            trigger: "a:contains('Large Meeting Table')",
```

Comment:
I think this can fix your test
```suggestion
            trigger: "a:contains('Chair floor protection')",
```

Reply:
Yeah, thanks.